src/main/java/com/odysseusinc/arachne/datanode/security/AuthenticationTokenFilter.java

```diff
@@ -64,15 +59,14 @@ public void doFilter(
         try {
 
             authenticationService.authenticate(authToken, httpRequest);
-            chain.doFilter(request, response);
         } catch (AuthenticationException | AuthException | org.ohdsi.authenticator.exception.AuthenticationException ex) {
-            ((HttpServletResponse) response).setStatus(HttpServletResponse.SC_UNAUTHORIZED);
-            JsonResult<Boolean> result = new JsonResult<>(JsonResult.ErrorCode.UNAUTHORIZED);
-            result.setResult(Boolean.FALSE);
-
-            response.getOutputStream().write(objectMapper.writeValueAsString(result).getBytes());
-            response.setContentType("application/json");
+            if (log.isDebugEnabled()) {
+                log.debug("Authentication failed", ex);
+            } else {
+                log.error("Authentication failed: {}", ex.getMessage());
```

Comment:
Error line is printed twice per a single request with improper token (`https://localhost:8880/api/v1/build-number`):
```
2019-08-16 10:18:04.849 ERROR 13432 --- [nio-8880-exec-9] c.o.a.d.s.AuthenticationTokenFilter      : Authentication failed: Expired or invalid JWT token
2019-08-16 10:18:04.850 ERROR 13432 --- [nio-8880-exec-9] c.o.a.d.s.AuthenticationTokenFilter      : Authentication failed: Expired or invalid JWT token
```

Reply:
First time filter triggered by OPTIONS and the second by GET